package.json

```diff
@@ -5,11 +5,11 @@
   "license": "MIT",
   "main": "index.js",
   "engines": {
-    "node": "^14.15.0"
+    "node": ">=14.15.0"
   },
   "scripts": {
     "start": "node server.js",
-    "start:dev": "hof-build watch",
+    "start:dev": "hof-build watch --env",
```

Comment:
I would just add a script called `dev` with --env
some devs don't use .env files like me and you